src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/IAppxManifestProperties.cs

```diff
@@ -4,7 +4,7 @@
 
 using System;
 using System.Runtime.InteropServices;
-using static Microsoft.Plugin.Program.Programs.UWP;
+using HRESULT = Wox.Plugin.Common.Win32.HRESULT;
```

Comment:
Feedback from above re: namespace

Reply:
Let's remove this assignment in the using as we don't have it on other pkaces where we use this enum.

Reply:
We do in AppxPackageHelper.cs but ok